### ISSUE TITLE
Add Biome-BGC

### ISSUE THREAD

**Issue by mdietze** [PecanProject/pecan#300]
https://github.com/bpbond/Biome-BGC

**Comment by ashiklom** [comment]
Tagging @bpbond because he expressed an interest in tackling this at some point.

**Comment by bpbond** [comment]
Definitely. Not in the next two months, but after that, this would be a good task for me.
